tests/test_doc_examples.py

```diff
@@ -14,22 +15,37 @@ def _get_coding_blocks(nb_file):
 
 
 @pytest.mark.parametrize(
-    ('nb_file', 'idx_execute'),
+    ('nb_file', 'idx_execute', 'needs_expyre'),
```

Comment:
@simonwengert what do you think about always executing all cells? In that case we can remove "idx_execute" option here?

Reply:
The idea here was that some examples contain code-blocks that are not intended to be executed as python code (`cli` commands or `conda` installations, see https://libatoms.github.io/workflow/examples.smiles.html). If we want to integrate these examples in the auto-testing at some point, such code blocks can be skipped then easily. Although, we do not make use of this option, I think it's good to have it available.